foreman/data_refinery_foreman/surveyor/management/commands/survey_all.py

```diff
@@ -70,3 +77,17 @@ def handle(self, *args, **options):
                             surveyor.survey_sra_experiment(accession)
                     except Exception as e:
                         logger.exception(e)
+
+        if options["accession"]:
```

Comment:
This is just a copy paste of the code right above it, you should functionalize this